src/frontend/src/pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModalContainer.tsx

```diff
@@ -26,7 +27,8 @@ export interface FormInput {
 const StageGateWorkPackageModalContainer: React.FC<StageGateWorkPackageModalContainerProps> = ({
   wbsNum,
   modalShow,
-  handleClose
+  handleClose,
+  hideStatus
```

Comment:
you can set hideStatus to false by default here so you don't have to have the undefined check below